Extensions/pokes.css

```diff
@@ -1,73 +1,215 @@
 }
 
-#xkit-loading_pokemon {
+/*
+ * Pokémon list style 
+*/
+.xkit-pokes-lightbox {
+	position: fixed;
+	top: 0px;
+	bottom: 0px;
+	right: 0px;
+	left: 0px; 
+	z-index: 15000;
+	overflow: hidden;
+	background-color: rgba(25,25,25,.92);
+}
+.gender {
+	display: none;
+	float: right;
+	font-family: "segoe ui symbol";
```

Comment:
@Wolvan I originally set a font because the female/male and genderless icons ended up positioned differently (and badly) in the default font for some reason, it was very strange.  If it's no longer doing that, there's no need for the font - it was meant as a very temporary measure anyway that I never got around to fixing